### PR TITLE
fix: Add Chromium Edge Support

### PR DESCRIPTION
**Pull request by nextux**
Add in support for Microsoft's Chromium based Edge browser.  Works just like any other Chrome based browser.

I have not added any download link to Edge, as Microsoft would generally include this browser as packaged inside Windows 10 and 11, and should be keeping it updated via Windows Update.

### CODE REVIEW

File: packages/suite-data/src/browser-detection/index.ts [hunk 1]
```diff
@@ -107,6 +107,11 @@ window.addEventListener('load', () => {
             version: 69,
             mobile: true,
         },
+        {
+            name: 'Edge',
+            version: 96,
+            mobile: true,
```
Comment (prusnak) [inline packages/suite-data/src/browser-detection/index.ts]: Have you tested Edge works with Trezor on Android? (that what mobile=true means)

Comment (nextux) [reply packages/suite-data/src/browser-detection/index.ts]: Apologies for the oversight here.  Checking on my Pixel 6, it appears the WebUSB is not working with Edge.  I have changed it to false in a further commit.

**Comment by tomasklim** [comment]
@matejkriz @prusnak https://github.com/trezor/trezor-suite/issues/4538#issuecomment-975377768 
Do we want to start supporting Edge now? If yes, we should also start supporting Opera, because it is also based on Chromium. 

We support Chrome from version 69, so we can support Edge and Opera versions which are based on Chromium >= 69

**Comment by prusnak** [comment]
> Do we want to start supporting Edge now?

I think it's bad idea to start supporting browsers we don't test in our QA. Each browser has it's quirks and especially with WebUSB it's super tricky.

We should think about which extra browsers we could try to support and why. My quick assesment:
* Edge - we could try to support this as it makes around 12% of installations worldwide
* Brave - we could try to support this as it is a privacy oriented browser and probably lot of our users use Brave for this reason
* other Chromium forks seem quite irrelevant at the moment: [src](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Summary_tables)

So the question really is - do the reasons above justify adding a browser into our default QA pipeline (basic WebUSB test)? If yes, then let's add them. This is for @tsusanka to decide.

**Comment by nextux** [comment]
Using Brave with my test suite, it appears to be working out of the box.  The user agent for the Brave browser I have installed is...

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36

According to https://www.ctrl.blog/entry/brave-user-agent-detection.html it appears as though Brave removed their component of the user agent string since around April 2016.

My reasoning for Edge is, as it is based on Chromium, I no longer really have a need to install Chrome on my Windows 11 machine.  Others may have a similar belief too.

**Comment by tsusanka** [comment]
My take on this: As we want to primarily focus on desktop app, I would not add support for more browsers at the moment.

That being said, I suggest to replace the current restriction

image

with a warning. We can even use the current Suite notification banners to inform about that.

So in other words, we will not actively support it, but in case it works out of the box, I do not see a reason to restrict the user. Any objections to that?

**Comment by nextux** [comment]
I think this might be the best way forward.  Agree.

**Comment by tomasklim** [comment]
Will be part of https://github.com/trezor/trezor-suite/issues/4750

**Pull request closed without merging**